src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -685,8 +685,8 @@ private void PreProcessUserFunctionStatement(ParseNode node)
 
                 // build default dummy function to be used when this is a LOCK:
                 currentCodeSection = userFuncObject.GetUserFunctionOpcodes(0);
-                AddOpcode(new OpcodeArgBottom());
-                AddOpcode(new OpcodePush("$" + userFuncObject.ScopelessIdentifier)).Label = userFuncObject.DefaultLabel;
+                AddOpcode(new OpcodeArgBottom()).Label = userFuncObject.DefaultLabel;;
```

Comment:
Correct.  That was the point.
The bug I was fixing is that it was jumping to the wrong line.  It's *supposed* to jump to the ArgBottom.  That's the first line of the routine.  By skipping past it before, it was causing the stack to not consume the argbottom mark.  Over time that clogged the stack with argbottoms that weren't being popped, each time you did a manual `run` commnd from the interpreter it left one behind for each lock steering and lock throttle in the programs (this bit of code is only run at the top of a new program context, so this only happened once per *manually* typed run command.  So it took time to build up enough cruft in the stack to notice the problem.